tests/perf-system/analyzer/analyzer.py

```diff
@@ -316,7 +316,7 @@ def plot_latency_distribution(self, ms_separator: float, highest_vals=15):
 
 
 def get_df_from_parquet_file(input_file: str):
-    return pd.read_parquet(input_file, engine="fastparquet")
+    return pd.read_parquet(input_file)

```

Comment:
For analysis, stopped using `fastparquet` for reading and instead using the default engine (`pyarrow`). Rough tests suggests that's 3x faster for the large files we're using. Still ~5s per file, which seems unreasonably slow, but I haven't found a faster path.